action.yml

```diff
@@ -20,5 +20,5 @@ inputs:
     required: false
     # Defaults to retaining all tagged images
 runs:
-  using: 'node12'
+  using: 'node18'
```

Comment:
This [doesn't exist](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runsusing).

The GitHub team [expects to eventually release a Node.js 20 runtime](https://github.com/actions/runner/issues/2255#issuecomment-1332128920) (`using: 'node20'`).

Reply:
@stuft2 👆

Reply:
I just pushed a commit reverting to node 16.